lib/docker2aci.go

```diff
@@ -169,22 +169,27 @@ func SquashLayers(images []acirenderer.Image, aciRegistry acirenderer.ACIRegistr
 	squashedFilename := getSquashedFilename(parsedDockerURL)
 	squashedImagePath := path.Join(outputDir, squashedFilename)
 
-	squashedImageFile, err := os.Create(squashedImagePath)
+	squashedTempFile, err := ioutil.TempFile(outputDir, "squashedFilename-")
 	if err != nil {
 		return "", err
 	}
-	defer squashedImageFile.Close()
+	defer squashedTempFile.Close()
+	squashedTempFilePath := filepath.Join(outputDir, squashedTempFile.Name())
```

Comment:
Is `filepath.Join` really needed?
squashedTempFile.Name() should just contain the outputDir.

I think that with this, if outputDir is `/tmp/something/` squashedTempFile will become `/tmp/something/tmp/something/filename`